sdkproject/Assets/Mapbox/Unity/MeshGeneration/Modifiers/ModifierBase.cs

```diff
@@ -4,8 +4,36 @@ namespace Mapbox.Unity.MeshGeneration.Modifiers
 	using System;
 	using Mapbox.Unity.MeshGeneration.Data;
 
+
```

Comment:
Move MapboxDataProperty to it's own file. Also, namespace for MapboxDataProperty should not be Mapbox.Unity.MeshGeneration.Modifiers